projectq/ops/_gates.py

```diff
@@ -137,6 +152,25 @@ def matrix(self):
 Swap = SwapGate()
 
 
+class SqrtSwapGate(BasicMathGate):
+    """ Square-root Swap gate class """
+    def __init__(self):
+        BasicMathGate.__init__(self, lambda x, y: (y, x))
```

Comment:
This is a left over from the Swap gate and needs to be removed (as it is not a BasicMathGate)